lib/hyrax/transactions/steps/save.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'dry/monads'
+
+module Hyrax
+  module Transactions
+    module Steps
+      ##
+      # Saves a given work, returning a Result (Success|Failure)
```

Comment:
Are `Success` and `Failure` name spaced? I don't know if that might be helpful to include an @see reference to their namespace location.

Reply:
it seems to be conventional in `dry-monads` to use just `Success` and `Failure`.

`include Dry::Monads[:result]` creates an alias for each in the local namespace: htps://github.com/dry-rb/dry-monads/blob/master/lib/dry/monads/result.rb#L296-L299

i'd welcome documentation improvements, especially if they can be applied to steps and transactions consistently.

Reply:
I'll take a look at the Yard output to see how it resolves.

Reply:
Yard does not resolve the constant as something visible in Hyrax's documentation. I think morning Jeremy is ready to resolve what evening Jeremy raised.